src/economy/economy.cpp

```diff
@@ -2059,10 +2327,18 @@ void update_pop_consumption(sys::state& state, dcon::nation_id n, float base_dem
 	auto nation_rules = state.world.nation_get_combined_issue_rules(n);
 	bool nation_allows_investment = state.world.nation_get_is_civilized(n) && (nation_rules & (issue_rule::pop_build_factory | issue_rule::pop_expand_factory)) != 0;
 	for(auto p : state.world.nation_get_province_ownership(n)) {
+
+		float subsistence = local_subsistence_factor(state, p.get_province());
+		float subsistence_life = std::clamp(subsistence, 0.f, subsistence_score_life);
+		subsistence -= subsistence_life;
+		float subsistence_everyday = std::clamp(subsistence, 0.f, subsistence_score_everyday);
+		subsistence -= subsistence_everyday;
+		float subsistence_luxury = std::clamp(subsistence, 0.f, subsistence_score_luxury);
+
 		for(auto pl : state.world.province_get_pop_location(p.get_province())) {
 			auto t = pl.get_pop().get_poptype();
 			assert(t);
-			auto total_budget = pl.get_pop().get_savings();
+			auto total_budget = pl.get_pop().get_savings() * 0.8f;
```

Comment:
is there a reason why you are having pops spend only 80% here? Unless there are other changes, pop income simply overwrites the old values, so the remainder will just be lost. Secondly, even if you have changed that, it still converges to pops spending exactly their income each day, the only difference being a slight lag until it reaches that point.

Reply:
To smooth out spendings a bit

Reply:
Ok, but there are places in the code where I have probably assumed that pop savings == pop income after the end of the day (which was the case before). You may want to search the files for `et_savings` and give every old use of this value an inspection